Day1/day1_p2.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const fs = require('fs');
+const arr = fs
+  .readFileSync('/Users/Elena/AdventOfCode/Day1/input_01.txt', 'utf-8')
+  .split('\n')
+  .map(x => parseInt(x));
+
+const searchThreeNumbers = arr => {
+  for (let i = 0; i <= arr.length; i++) {
+    for (let b = i + 1; b <= arr.length; b++) {
+      for (let j = i + 2; j <= arr.length; j++) {
+        if (arr[i] + arr[b] + arr[j] === 2020) {
+          const array = [arr[i], arr[b], arr[j]];
+          return array;
+        }
+      }
+    }
+  }
+};
+searchThreeNumbers(arr);
```

Comment:
The same question as in the previous file.
Why do you run the function here?
When you `require` this module in a test, for example, this code will read the file, find the answer, ignore it and then pass the function to the code that `require`d it.

You need to split the function that finds the answer and the code that uses it, like the code that reads the file, executes the function, and displays the answer. Similarly to what you did in `Day2/day2.js`

By the way, how do you know what the answer is? You don't display it anywhere. It's ignored.